java/gandiva/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(GANDIVA_PROTO_DIR ${CMAKE_CURRENT_SOURCE_DIR}/proto)
 get_filename_component(GANDIVA_PROTO_FILE_ABSOLUTE ${GANDIVA_PROTO_DIR}/Types.proto
                        ABSOLUTE)
 
+set(Protobuf_USE_STATIC_LIBS ON)
```

Comment:
Could you specify this in build script (`ci/scripts/java_jni_build.sh`)?
Some users may want to use `libprotobuf.so`.

Reply:
Done.